orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/WaitForDownInstanceHealthTask.groovy

```diff
@@ -28,8 +28,10 @@ class WaitForDownInstanceHealthTask extends AbstractWaitForInstanceHealthChangeT
       return true
     }
 
+    boolean allAreUnknown = healthProviders.every {it.state == 'Unknown'}
```

Comment:
I'm not sure we should be using this flag here unless the 'only consider platform health' option is selected